### PR TITLE
Fix width

### LINKED ISSUE #86: Purchase button is misaligned on desktop
### General information

same as previous issue

### Issue description
screen shot 2017-03-04 at 14 14 42

Fix
```
@media (min-width: 768px) {
    #braintree-submit-button {
        margin-left: 3px;
    }
}
```

Post-fix
screen shot 2017-03-04 at 14 18 50

### PR DESCRIPTION
**Pull request by crookedneighbor**
### Summary

Fixes #86 

Removes 9x% widths in favor of 💯 % length so that alignment is easier to manage.

Also adjusts the positioning of some elements to accommodate the full width look.

### Checklist

- [x] Added a changelog entry
- [x] Ran unit tests

### CODE REVIEW

File: src/scss/main.scss [hunk 1]
```diff
@@ -590,15 +583,14 @@ $loader-scale-duration: 300ms;
...
   cursor: pointer;
   display: flex;
   justify-content: space-between;
   margin: 0 auto;
   padding: 14px 10px;
   position: relative;
-  transition: opacity 300ms, box-shadow 300ms, width 200ms $curve-easeOutBack, border-color 300ms, border-radius 300ms;
-  width: 99%;
+  transition: opacity 300ms, width 200ms $curve-easeOutBack, border-color 300ms, border-radius 300ms;
+  width: 98%;
```
Comment (intelliot) [inline src/scss/main.scss]: This is 98% now? Not sure which part of it this affects.

**Comment by intelliot** [comment]
Here's an image diff.. looks like a good improvement.

visual diff download

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ UNRELEASED
 ---------
 - Fix typo in russian translation
 - Update browser detection library to 1.4.0
+- Fix width errors where drop-in was not aligned with other elements on merchant page
```
Comment (lilaconlee) [inline CHANGELOG.md]: drop-in -> Drop-in

File: src/scss/main.scss (hunk 1 above)
Comment (lilaconlee) [inline src/scss/main.scss]: I think it'll just make the methods a bit smaller unless it's the active method.

File: src/scss/main.scss [hunk 3]
```diff
@@ -799,11 +799,11 @@ $loader-scale-duration: 300ms;
 }
 
 /* =========================================================================
-   Remove this when we have a payment methods width 100%  width    
+   Remove this when we have a payment methods width 100%  width
```
Comment (lilaconlee) [inline src/scss/main.scss]: Not necessarily part of this PR, but is this something we intend to do?

**Pull request merged**